clients/ITAC_WinClient/Option.xaml.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using ITAC_WinClient.Models;
+
+namespace ITAC_WinClient
+{
+    /// <summary>
+    /// Interaction logic for Option.xaml
+    /// </summary>
+    public partial class Option : Window
+    {
+        //func validate dugit in fields
+
+        private void ValidateIsNumeric(TextCompositionEventArgs e)
+        {
+            int result;
+
+            if (!(int.TryParse(e.Text, out result)))
```

Comment:
I think you can declarate you variable inside TryParse call, and if you don't use variable you can use '_' instead of name:
int.TryParse(e.Text, out int _)
Why you don't use function like this:
private bool IsNumber(string text) => int.TryParse(text, out int _)